### DISCUSSION TITLE
pretrained synth morph 3d

### DISCUSSION THREAD

**Discussion by jakubMitura14** [voxelmorph/voxelmorph#549] [Q&A]
Hello are there any weights that could be downloaded for registration of 3D images (CT MRI) for synthmorph?

**Comment by mu40** [comment]
Thank you for your question! While we do not have weights specific to CT-MRI registration, we now have a tutorial that demonstrates how to enhance soft-tissue contrast by clipping the CT intensities, which can improve multi-modal registration with SynthMorph. You can find it in the "Demos" section at [synthmorph.io](https://synthmorph.io).
